docs/content/security.md

```diff
@@ -133,10 +133,14 @@ must be provided on startup. The authorization policy must be structured as foll
 # system.authz as follows:
 package system.authz
 
-default allow = false  # Reject requests by default.
+# Reject requests by default.
+default allow = {
+	"allowed": false,
+	"reason": "unauthorized resource access"
+}
 
-allow {
-  # Logic to authorize request goes here.
+allow = {"allowed": true} {
+	# Logic to authorize request goes here.
 }
 ```
```

Comment:
Could you add a note re: boolean responses still being OK? Since we had previously removed them completely, you've done all the work to update the document, but I think we should still mention that a true/false result is supported.

Maybe we could keep most of the examples as-is, but add another section mentioning the fancier, new `deny`s with a reason? What do you think?